service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandlerTest.java

```diff
@@ -248,6 +253,29 @@ void shouldNotifyNonHmctsAdminOnC2Upload() throws IOException, NotificationClien
                 eq(C2_UPLOAD_NOTIFICATION_TEMPLATE), eq("hmcts-non-admin@test.com"), eq(c2Parameters), eq("12345"));
         }
 
+        @Test
+        void shouldNotifyCtscAdminOnC2UploadWhenCtscIsEnabled() throws IOException, NotificationClientException {
+            CallbackRequest callbackRequest = appendEnableCtscOnCallback(callbackRequest());
+            CaseDetails caseDetails = callbackRequest.getCaseDetails();
+
+            given(idamApi.retrieveUserInfo(AUTH_TOKEN)).willReturn(
+                UserInfo.builder().sub(CTSC_INBOX).roles(LOCAL_AUTHORITY.getRoles()).build());
+
+            given(ctscEmailLookupConfiguration.getEmail()).willReturn(CTSC_INBOX);
+
+            given(inboxLookupService.getNotificationRecipientEmail(caseDetails, LOCAL_AUTHORITY_CODE))
+                .willReturn(LOCAL_AUTHORITY_EMAIL_ADDRESS);
+
+            given(c2UploadedEmailContentProvider.buildC2UploadNotification(caseDetails))
+                .willReturn(c2Parameters);
```

Comment:
some of this could be extracted to a before

Reply:
Should we also group ALL 'send to ctsc' tests?